pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -209,6 +210,7 @@ func (pl *InterPodAffinity) getIncomingAffinityAntiAffinityCounts(ctx context.Co
 			affinityCountsList[k] = affinity
 			antiAffinityCountsList[k] = antiAffinity
 		}
+		return nil
 	}
 	pl.parallelizer.Until(ctx, len(allNodes), processNode, pl.Name())
```

Comment:
Check the error from Until, although I know it shouldn't return in any case, it could prevent the future mistake